src/components/AdmissionForm/index.tsx

```diff
@@ -384,7 +384,7 @@ const AdmissionForm = () => {
         <label className="ms-2 text-sm font-medium text-gray-900 dark:text-gray-300">
           I agree with the{" "}
           <a
-            href="#"
+            href="/terms_conditions"
```

Comment:
In place of anchor tag `a`, use `Link`
```suggestion
            <Link
            to="/terms_conditions"
            className="text-blue-600 hover:underline dark:text-blue-500"
            >
```